frontend/src/main/scala/Directors.scala

```diff
@@ -0,0 +1,50 @@
+package eoi
+package frontend
+
+import main.scala.DirectorsRowInput
+
+import scala.scalajs.js.Date
+import scala.scalajs.js
+import JsObjectAdapter.ops._
+import eoi.frontend.Directors.DirectorsRow
+
+import scala.scalajs.js.JSConverters._
+import scala.scalajs.js.annotation.JSExportTopLevel
+
+class Directors (
+  config: Configuration
+                ) extends js.Object {
+
+  import ClassOneFrontend.c1ResultLikeAdapter
+
+  def calculate(
+                 from: Date,
+                 to: Date,
+                 rows: js.Array[DirectorsRow],
+                 appropriatePersonalPensionScheme: Option[Boolean]
+               ): js.Object = config.calculateDirectors(
+    from,
+    to,
+    rows.map(row => DirectorsRowInput(row.category.head, row.grossPay, row.id)),
+    appropriatePersonalPensionScheme
+  ).toJSObject
+
+  def getTaxYearsWithOptions: js.Array[String] =
+    config.classOne.keys.map(_.toString).toJSArray
+
+  def isAppropriatePersonalPensionSchemeApplicable(on: Date) =
+    on.getYear < 2016
+
+
+}
+
+object Directors {
+
+  @JSExportTopLevel("ClassOneRow")
```

Comment:
maybe this should be `"DirectorsRow"` given we already have a top level export with this name

Reply:
oh yeah, whoops - it's cos I copy and pasted it and forgot to change it

Reply:
fixed it now

Reply:
I think the argument with the name is optional and it should default to the class name.